### ISSUE TITLE
Callling QGuidoPainter::setGuidoPageFormat causes an error: unexpected $end, expecting STARTSEQ

### ISSUE THREAD

**Issue by yurivict** [grame-cncm/guidolib#37]
In environments/Qt/examples/example1, I added ```setGuidoPageFormat```:
```
        QGuidoPainter * p = QGuidoPainter::createGuidoPainter();
        p->setGuidoPageFormat({300/*width*/, 400/*height*/, 10/*marginleft*/, 10/*margintop*/, 10/*marginright*/, 10/*marginbottom*/});
```

After this, it began to print:
```
error line: 1 col: 0: syntax error, unexpected $end, expecting STARTSEQ
```

**Comment by dfober** [comment]
it looks like _setGuidoPageFormat_ triggers an evaluation of the guido code, which is empty at this step. Note that despite the error message, the exported image is correct. 
Moving the function call after the _setGMNCode_ will suppress the warning.
Note that I don't feel happy with the Qt library API and implementation: too much side effects, not a functional design.

**Comment by yurivict** [comment]
> Note that I don't feel happy with the Qt library API and implementation: too much side effects, not a functional design.

I needed to find a way to draw some notes for my own little app to play with sounds, and guidolib is the only solution I found. There is MusicXML, but it doesn't have a (free and accessible) rendering library. So far, guidolib with Qt rendering worked well for me, thank you for working on it!

I liked GuidoLib also because it is actually human-readable and writable, unlike MusicXML.

I also found that at least one element, tablature, isn't covered by guidolib (?).

I also created the FreeBSD port for it: https://bugs.freebsd.org/bugzilla/show_bug.cgi?id=221838
